R/Sound_DMR.R

```diff
@@ -570,17 +570,17 @@ run_binomial <- function(LM, i = int, formula,
 #' @export
 #'
 
-run_model <- function(data, i, Output_Frame, formula, model_type,
+run_model <- function(data, i, Out_in, formula, model_type,
```

Comment:
nit: to me  `Out_in` is a mysterious argument name? Not necessarily better or worse than `Output_Frame` i suppose.  Since the doc string is set up to inherit parameter description from `run_binomial` this change looks to create a mismatch such that you should update the argument name in that function as well.

Reply:
This was changed because there were multiple uses of Output_Frame frame both as a local and global variable which was causing an ambiguous misdirect which is why Jack changed it from Output_Frame. I'll change it..